sktime/transformations/series/tests/test_kalman_filter.py

```diff
@@ -200,8 +200,8 @@ def init_kf_filterpy(measurements, adapter, n=10, y=None):
 
 
 @pytest.mark.skipif(
-    not _check_soft_dependencies("pykalman", severity="none"),
-    reason="skip test if required soft dependency pykalman not available",
+    not _check_soft_dependencies("pykalman-bardo", severity="none"),
```

Comment:
```suggestion
    not _check_soft_dependencies("pykalman-bardo", package_import_alias={"pykalman-bardo": "pykalman"}, severity="none"),
```

I think this is required, plus black formatting.